app.js

```diff
@@ -0,0 +1,52 @@
+// JS2 week1 - Browser environment, DOM manipulation, DOM event listeners
+
+//  create a form to insert a quiz question. It should take a question and 4 options for answers.
+//There should be a way to mark which answer is correct in the form, but you should only be able to select one correct answer.
+
+//  implement one of the two (or both!):
+
+// a. add a button that randomizes the order of the 4 option inputs once they have been filled in.
+
+// b. color the input for the "correct" answer option in green and the "wrong" ones in red. Make sure it's still readable.
+
+//  optional bonus: make sure the question is not longer than 140 characters.
+
+function randomizeOptions() {
+  var optionsContainer = document
```

Comment:
you can use `const` for that, variables declared with `var` have a different scope, wich may sometimes cause problems. Always use `const` for variables that you will not reaasign and `let` for the ones that will chang ethe value